### PR TITLE
Misc: Add missing copyright headers

### PR DESCRIPTION
**Pull request by linusg**
Please don't merge right away, let's give these people some time to confirm they want to be copyright holders of the files they created, otherwise I'll change it to the generic `Copyright (c) YYYY, the SerenityOS developers.`.

---

- @awesomekling 
  - `AK/Memory.h`
  - `Applications/Browser/WindowActions.cpp`
  - `Applications/Browser/WindowActions.h`
  - `Demos/Mouse/main.cpp`
  - `Userland/disasm.cpp`
- @itamar8910 
  - `Kernel/Ptrace.cpp`
- Written by @predmond but no header with their info already exists
  - `Meta/Lagom/Fuzzers/FuzzJs.cpp`
- ~~@alimpfard~~
  - ~~`Userland/test-crypto.cpp`~~
- @howar6hill 
  - `AK/Tests/TestStringUtils.cpp`

---

I'd also like to propose a script that runs on CI and fails the build if `.{cpp,h}` files in certain directories don't contain a copyright header. (yes, I like automation)

### CODE REVIEW

**Comment by alimpfard** [comment]
That error-if-copyright-missing script sounds nice.

I'm also just fine with "The SerenityOS developers." on test-crypto (is it even sensible to hold rights over a test suite?)

**Comment by linusg** [comment]
> is it even sensible to hold rights over a test suite?

maybe :)

most of the files in `Tests/Kernel` don't have a header, but on the other hand the ones in `AK/Tests` do.

File: Kernel/Ptrace.cpp [hunk 1]
```diff
@@ -1,3 +1,29 @@
+/*
```
Comment (itamar8910) [inline Kernel/Ptrace.cpp:1]: Thanks @linusg :)

**Comment by awesomekling** [comment]
A script sounds good :)

**Comment by awesomekling** [comment]
Okay let's merge this. If someone wants their copyright removed in the future, we can fix that then. :)

**Pull request merged**